app/api/client_resources.py

```diff
@@ -235,7 +244,7 @@ def resource_create(
                 "isBrowserShortcutEnabled": resource.is_browser_shortcut_enabled,
                 "remoteNetworkId": resource.remote_network_id,
                 "securityPolicyId": resource.security_policy_id,
-                "protocols": resource.protocols,
+                "protocols": resource.protocols.dict(by_alias=True),
```

Comment:
`resource.protocols` is a PyDantic object which is not JSON serializable.
We didnt have a test path here send sends a non-null value before so this never failed